Lib/tkinter/test/test_tkinter/test_widgets.py

```diff
@@ -940,7 +940,8 @@ def test_digits(self):
 
     def test_from(self):
         widget = self.create()
-        self.checkFloatParam(widget, 'from', 100, 14.9, 15.1, conv=float_round)
+        cvt = False if sys.platform == 'win32' else float_round  # 39107
```

Comment:
It should depend not on platform, but on Tk version. Use `get_tk_patchlevel()`.

Also, it would be better to use name `conv` instead of `cvt` for consistence.

Reply:
Agreed.  However, I could not fine that name in either tkinter or _tkinter, so I used the tk call used in idlelib.help_about.  I used an equality check.  But it would not work for an 8.6.11.  Is such possible?

Reply:
`get_tk_patchlevel` is defined in `tkinter.test.support` and is already imported in this file.

Reply:
Fixed.

Reply:
Serhiy, do you prefer passing False, as the ttk Scale test does, or 'noconv', as in #21715?
Do you want a comment added, such as in the other PR?
Please decide whether you want to close the other issue and PR or merge it and remove the test change here.
Either way, I would like to get this merged so we can routinely test tkinter and IDLE on Windows with 8.6.10 in place.